app/models/assignment.rb

```diff
@@ -0,0 +1,5 @@
+class Assignment < ApplicationRecord
+    belongs_to :school_class
+    belongs_to :teacher
+    belongs_to :lecture
```

Comment:
L'assignment quindi belongs alla lecture in quanto viene "dato" durante quella lezione?

Reply:
Si, l’ho pensato in quel modo. Se non convince si può anche togliere perchè non è necessario, ma secondo me ha senso per “tracciare” nel dettaglio quando il compito è stato assegnato

Reply:
Avrà anche una data di scadenza tipo, per sapere entro quando fare i compiti però